controllers/users.controller.js

```diff
@@ -2,7 +2,8 @@ import asyncDec from 'utils/asyncDecoration';
 import userCore from 'core/users.core';
 
 const getUserById = asyncDec(async (req, res) => {
-  const dataResponse = await userCore.findUserById(req.params.userId);
+  const userId = (req.params.userId === 'me') ? req.uid : req.params.userId;
+  const dataResponse = await userCore.findUserById(userId);
```

Comment:
Remove and only use req.uid

Reply:
https://github.com/EasyEventV2/ee2_api/pull/13/commits/7a32e97e8ab7d658de30835a436504293227f42b